src/main/groovy/de/tracetronic/jenkins/plugins/ecutestexecution/steps/GenerateReportsStep.groovy

```diff
@@ -147,8 +147,9 @@ class GenerateReportsStep extends Step {
 
             reportIds.each { reportId ->
                 listener.logger.println("- Generating ${this.generatorName} report format for report id ${reportId}...")
-                result = apiClient.generateReport(reportId, generationOrder)
-                listener.logger.println(result.toString())
+                GenerationResult generationResult = apiClient.generateReport(reportId, generationOrder)
+                listener.logger.println(generationResult.toString())
+                result.add(generationResult)
             }
```

Comment:
there was an push missing...

Reply:
Better, but still the logformat some makes is hard to get a quick overview

```
- Generating ATX report format for report id 34b4ff75-7a28-4898-986b-d40c469f3f1e...
-> result: FINISHED
-> message: 
-> reportOutputDir: < Workspace Path >\TestReports\test_2024-02-22_174023\ATX7
```

* Maybe we can print the (releative) Path to the report/trf instead of the id
* We might want to change the toString Method of the GenerationResult 
  * e.g do not print `message` if its empty